test/ode_solvers_test.cpp

```diff
@@ -0,0 +1,462 @@
+/*
+GeomScale Project
+
+Copyright (c) 2020
+  Vissarion Fisikopoulos
+  Apostolos Chalkis
+  Elias Tsigaridas
+  Marios Papachristou
+
+Contributed and/or modified by Marios Papachristou,
+as part of Google Summer of Code 2020 program.
+
+VolEsti is free software: you can redistribute it and/or modify it
+under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or (at
+your option) any later version.
+
+VolEsti is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+See the file COPYING.LESSER for the text of the GNU Lesser General
+Public License.  If you did not receive this file along with HeaDDaCHe,
+see <http://www.gnu.org/licenses/>.
+*/
+
+#include "doctest.h"
+#include "Eigen/Eigen"
+#include "ode_solvers.h"
+#include <iostream>
+#include <cmath>
+#include <functional>
+#include <vector>
+#include <unistd.h>
+#include <string>
+#include <typeinfo>
+
+#include "random.hpp"
+#include "random/uniform_int.hpp"
+#include "random/normal_distribution.hpp"
+#include "random/uniform_real_distribution.hpp"
+#include "random_walks/random_walks.hpp"
+
+#include "known_polytope_generators.h"
+
+#include "volume/volume_sequence_of_balls.hpp"
+#include "volume/volume_cooling_gaussians.hpp"
+#include "volume/volume_cooling_balls.hpp"
+
+#include "exact_vols.h"
+#include "generators/known_polytope_generators.h"
+
+
+template <typename NT>
+void test_euler(){
+    typedef Cartesian<NT>    Kernel;
+    typedef typename Kernel::Point    Point;
+    typedef std::vector<Point> pts;
+    typedef std::function<Point(pts, NT)> func;
+    typedef std::vector<func> funcs;
+    typedef boost::mt19937    RNGType;
```

Comment:
this is unused

Reply:
the same holds for all the functions in this file

Reply:
Thank you. I had v-polys in the previous version of the tests w/ the previous versions of the project. 
I removed the redundant typedefs.